lib/parser/parsing.h

```diff
@@ -73,6 +74,7 @@ class Parsing {
 private:
   Options options_;
   CookedSource cooked_;
+  const SourceFile *sourceFile_{nullptr};
```

Comment:
Please add a comment that identifies the owner of this storage.

Maybe it doesn't need to be a permanent part of the state here.  What if `Prescan()` were to return this pointer instead?

Reply:
OK, I've changed `Prescan()`.